charts/base-cluster/values.yaml

```diff
@@ -28,6 +28,10 @@ global:
       condition: "{{ .Values.ingress.enabled }}"
       additionalLabels:
         app.kubernetes.io/component: ingress
+    tetragon:
+      condition: "{{ .Values.tetragon.enabled }}"
+      additionalLabels:
+        app.kubernetes.io/component: tetragon
```

Comment:
I would rather put this in `kube-system`

Reply:
done